### PR TITLE
Document all of rpmdeps options

### PR DESCRIPTION
**Pull request by soig**
This will update the man page in order to describe all rpmdeps options.

### CODE REVIEW

File: doc/rpmdeps.8 [hunk 1]
```diff
@@ -19,6 +19,35 @@ for the set of \fIFILE\fR arguments.
...
+Print the enhances
+.TP
+\fB--conflicts\fR
+Print the conflicts
+.TP
+\fB--obsoletes\fR
+Print the obsoletes
+.TP
+\fB--alldeps\fR
+Print the alldeps
```
Comment (pmatilai) [inline doc/rpmdeps.8:49]: There's no such thing as "the alldeps". 
--alldeps prints out all the dependencies in one go, using an entirely different formatting.

**Comment by soig** [comment]
Patch updated (semi automatic description of options is not always a good idea :-( )

**Comment by soig** [comment]
2nd commit has been droped

**Comment by pmatilai** [comment]
Thanks for the patch!

**Pull request merged**